roughpy_jax/__init__.py

```diff
@@ -0,0 +1,132 @@
+import numpy as np
+import jax
```

Comment:
I think the correct way to handle the JAX dependency is to try-import here and emit an import exception with instructions to install JAX to use the roughpy jax features.